entrypoint/clusterrestore/clusterrestore_suite_test.go

```diff
@@ -0,0 +1,87 @@
+package clusterrestore
+
+import (
+	"flag"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/rancher/distros-test-framework/config"
+	"github.com/rancher/distros-test-framework/pkg/aws"
+	"github.com/rancher/distros-test-framework/pkg/customflag"
+	"github.com/rancher/distros-test-framework/pkg/qase"
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var (
+	qaseReport = os.Getenv("REPORT_TO_QASE")
+	flags      *customflag.FlagConfig
+	kubeconfig string
+	cfg        *config.Product
+	cluster    *shared.Cluster
+	awsClient  *aws.Client
+)
+
+func TestMain(m *testing.M) {
+	var err error
+	flags = &customflag.ServiceFlag
+	flag.Var(&flags.Destroy, "destroy", "Destroy cluster after test")
+	flag.StringVar(&flags.S3Flags.Bucket, "s3Bucket", "distros_qa", "s3 bucket to store snapshots")
+	flag.StringVar(&flags.S3Flags.Folder, "s3Folder", "snapshots", "s3 folder to store snapshots")
+	flag.Parse()
+
+	cfg, err = config.AddEnv()
+	if err != nil {
+		shared.LogLevel("error", "error adding env vars: %w\n", err)
+		os.Exit(1)
+	}
+
```

Comment:
Should add validations for failing the tests if unsupported params are provided

Reply:
which unsupported params we have here?

Reply:
Like passing `profile: cis` or `selinux: true` on the server_flags

Reply:
true adding